samples/msal-angular-v3-samples/angular15-sample-app/package.json

```diff
@@ -35,7 +36,7 @@
     "@angular/compiler-cli": "^15.1.0",
     "@types/jasmine": "~4.3.0",
     "jasmine-core": "~4.5.0",
-    "jest": "^29.4.3",
+    "jest": "^27.0.4",
```

Comment:
was this intentional?

Reply:
Same question here

Reply:
When I initially built this sample, I thought to use the latest version of Jest with the latest version of Angular. Unfortunately, there are some breaking changes with Jest@28, and since all our samples currently use 27, I thought to use 27 here until we can address those breaking changes in another PR at a later date.